cms/envs/common.py

```diff
@@ -311,6 +306,37 @@
 ############################ Modulestore Configuration ################################
 MODULESTORE_BRANCH = 'draft-preferred'
 
+MODULESTORE = {
```

Comment:
Note for reviewers: added this instead of just relying on LMS' configuration since Studio does not use the XML modulestore.